### DISCUSSION TITLE
Nanit Integration?

### DISCUSSION THREAD

**Discussion by apnelson13** [Sunoo/homebridge-camera-ffmpeg#1219] [Q&A]
Has anyone gotten this to work with a Nanit camera? I have seen some talented programmers who reverse engineered much of the code and someone even got it to work on HA, but I am hoping to find a way to crack the camera URL and token so I can just use this plugin. So, anyone have any luck at this? 

Thanks,
Alex

**Comment by tjmehta** [comment]
you can with this:

https://gitlab.com/adam.stanek/nanit + mqtt
